docs/workloads/azure/data/getting_started/core_data_platform_deployment_azure.md

```diff
@@ -7,12 +7,82 @@ hide_table_of_contents: false
+|-----------------------------------------------|---------------------------------|
+| What framework should be used for the project?| infra                           |
+| Which type of infrastructure is required?     | data                            |
+
+The resulting configuration file named stacks.yml contains all of the configuration that was used to generate the projects,
+which means it can be used to produce the same project stack again.
+
+The CLI can be used with a configuration file to generate the Ensono Digital Stacks based projects using `stacks-cli scaffold`.
+
+```cmd
+stacks-cli scaffold -c ./stacks.yml
+```
+
+Open the project locally and push the generated project to target repository in `main/master` branch.
+
+## Step 2: Add Infrastructure pipeline in Azure Devops
+
+Open the project locally and create a new feature branch e.g.:
+
+```bash
+git checkout -b feat/infra-pipeline
+```
+
+YAML file containing a template Azure DevOps CI/CD pipeline for building and deploying the core infrastructure, named `air-infrastructure-data.yml` is configured in the `build/azDevOps/azurefolder`
+
+YAML file `air-infrastructure-data.yml` should be added as the definition for a new pipeline in Azure DevOps.
+
+1. Sign-in to your Azure DevOps organization and go to your project
+2. Go to Pipelines, and then select **New pipeline**
+3. Name the new pipeline, e.g. `amido.stacks-data-infrastructure`
+4. For the pipeline definition, specify the YAML file in the project repository feature branch (`air-infrastructure-data.yml`) and save
+5. The new pipeline will require access to any Azure DevOps pipeline variable groups specified in the pipeline YMAL. Under each variable group, go to 'Pipeline permissions' and add the pipeline.
+
+The `build/azDevOps/azure/network` folder contains a YAML file named `air-infrastructure-data-network.yml` that serves as a template for Azure DevOps CI/CD pipeline for networking and self-hosted agent infrastructure. If the client or user does not have their own network and self-hosted agent, they must establish the networking pipeline in Azure DevOps following the procedures outlined above. This setup should be completed before initiating the core infrastructure pipeline. The `build/azDevOps/azure/network` folder also includes another YAML file called `air-infrastructure-data-network-vars.yml` that contains the variables used in the networking pipeline. These variables must be updated as well as the project requirements.
+
+If the client/user has their own network and self-hosted agent, then the networking pipeline won't be required.
+
+## Step 3: Deploy Infrastructure in non-production environment
+
+Run the  pipeline configured in Step 2
+
+Running this pipeline in Azure DevOps will deploy the artifacts into the non-production (nonprod) environment. If successful, the generated resources will now be available in the nonprod Stacks environment.
+
```

Comment:
We have some variable groups that are listed as being added after core infra deployment. If so, we need a step here to say they need adding.

Reply:
updated as per feedback